lib/code_corps/accounts/accounts.ex

```diff
@@ -29,8 +34,107 @@ defmodule CodeCorps.Accounts do
   def create_from_github_changeset(struct, %{} = params) do
```

Comment:
This is not really part of this PR, but it is something that we've discussed before and I feel should be tackled.

`create_from_github_changeset/2` is public. The only place it's being called from publically is another public method in this module (`create_from_github/1`) and tests.

That means, it is in fact private and we made it public in order to test it.

The conventional approach if a private method is complex enough to warrant testing would be to extract the method's behavior into a submodule.

`Accounts.Changesets` would sound like a good candidate for that.